src/main/java/com/khutircraftubackend/services/registration/RegistrationSellerController.java

```diff
@@ -0,0 +1,34 @@
+package com.khutircraftubackend.services.registration;
```

Comment:
you probably wanted to move this class into `package com.khutircraftubackend.controllers;`

Reply:
Not at all. This package and classes in it would probably be better placed separately?